src/ServicePulse.Host/app/js/views/archive/view.html

```diff
@@ -11,7 +11,7 @@
         <div class="row" ng-show="!vm.loadingData">
             <div class="col-sm-12">
                 <div class="btn-toolbar">
-                    <button type="button" class="btn btn-default" confirm-message="Are you sure you want to un-archive selected archived messages?" confirm-click="vm.unarchiveSelected()" ng-disabled="vm.selectedIds.length == 0"><i class="fa fa-undo"></i> Unarchive {{vm.selectedIds.length | number}} selected</button>
+                    <button type="button" class="btn btn-default" confirm-title="Are you sure you want to un-archive selected archived messages" confirm-message="Are you sure you want to un-archive selected archived messages?" confirm-click="vm.unarchiveSelected()" ng-disabled="vm.selectedIds.length == 0"><i class="fa fa-undo"></i> Unarchive {{vm.selectedIds.length | number}} selected</button>
```

Comment:
@sergioc What kind of message should be here? The confirmation message is a 1 line only so repeating it in title and message looks bad.